Packs/AzureSQLManagement/Integrations/AzureSQLManagement/AzureSQLManagement_description.md

```diff
@@ -0,0 +1,21 @@
+In both options below, the [device authorization grant flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-device-code) is used.
+
+In order to connect to the Azure Network Security Group using either Cortex XSOAR Azure App or the Self-Deployed Azure App:
+1. Fill in the required parameters.
```

Comment:
How do we get each of these 3 params?

Reply:
I will show you, it written at the azure portal